BIPs/00batched/2023-W22/1-0xc38c5f97B34E175FFd35407fc91a937300E33860.json

```diff
@@ -0,0 +1,139 @@
+{
+  "version": "1.0",
+  "chainId": "1",
+  "createdAt": 1675891944772,
+  "meta": {
+    "name": "Transactions Batch",
+    "description": "",
+    "txBuilderVersion": "1.13.2",
+    "createdFromSafeAddress": "0xc38c5f97B34E175FFd35407fc91a937300E33860",
+    "createdFromOwnerAddress": ""
+  },
+  "transactions": [
+    {
+      "to": "0x5DbAd78818D4c8958EfF2d5b95b28385A22113Cd",
+      "value": "0",
+      "data": null,
+      "contractMethod": {
+        "inputs": [
+          {
+            "internalType": "address",
+            "name": "gauge",
+            "type": "address"
+          },
+          {
+            "internalType": "string",
+            "name": "gaugeType",
+            "type": "string"
+          }
+        ],
+        "name": "addGauge",
+        "payable": false
+      },
+      "contractInputsValues": {
+        "gauge": "0xb78543e00712C3ABBA10D0852f6E38FDE2AaBA4d",
+        "gaugeType": "EthereumSingleRecipientGauge"
```

Comment:
I wonder why this gauge didn't get included in the checkpointer-management payload (`BIPs/00batched/2023-W22/1-anySafeWillDo.json`)

Reply:
Not that it matters much, as adding gauges of this type should be rare